plugins/default-browser-emulator/lib/helpers/setScreensize.ts

```diff
@@ -17,15 +17,13 @@ export default async function setScreensize(
   if (emulator.browserEngine.isHeaded && viewport.screenWidth) {
     promises.push(
       page.devtoolsSession.send('Browser.getWindowForTarget').then(({ windowId, bounds }) => {
-        if (bounds.width === viewport.width && bounds.height === viewport.height) {
+        if (bounds.width === viewport.screenWidth && bounds.height === viewport.screenHeight) {
```

Comment:
This is a side note, but do you ever get confused by viewport.width and viewport.screenWidth? Should we rename viewport.width to viewport.browserWidth?